polling_stations/templates/fragments/cancelled_election.html

```diff
@@ -14,12 +14,11 @@ <h2>{% trans "Cancelled Election" %}</h2>
     {% else %}
         <p>{% blocktrans with cancelled_election_name=cancelled_election.name %}The poll for <strong>{{ cancelled_election_name }}</strong> has been cancelled.{% endblocktrans %}</p>
     {% endif %}
+    <p>To learn more, please visit <a href="https://whocanivotefor.co.uk/elections/{{ cancelled_election.slug }}/">Who Can I Vote For?</a></p>
```

Comment:
There's no `slug` property on `cancelled_election`, so this won't work. I think it would be better to direct them to `whocanivotefor.co.uk/elections/{{ postcode }}`. Of course this might be confusing with split postcodes, but it's probably better than nothing. If (as seems imminent) WCIVF gets an address picker, then we might need to be clever about sending a postcode or a uprn depending on whether the postcode is split or not. But maybe let's cross that bridge later.

The reason I don't think we can use an election slug is that there might be more than one, (eg local and mayor). For context: we arrive at the `cancelled_election` fragment if the `context["has_election"]` is false  ([link](https://github.com/DemocracyClub/UK-Polling-Stations/blob/7cc8d2fbf7282572992ed6ae7c88f3132bd2547b/polling_stations/templates/postcode_view.html#L36)) and there is a cancelled election ([link](https://github.com/DemocracyClub/UK-Polling-Stations/blob/7cc8d2fbf7282572992ed6ae7c88f3132bd2547b/polling_stations/templates/fragments/no_election.html#L4)). 

So we might want to think about handling the case where there is an uncancelled and a cancelled election by telling people about the cancelled election in a card below their polling station information. However I'm hesitant about that, as it feels like scope creep for WDIV.